src/o365/spo/commands/list/list-view-list.ts

```diff
@@ -0,0 +1,176 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+import { ContextInfo } from '../../spo';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+}
+
+class SpoListViewListCommand extends SpoCommand {
+  public allowUnknownOptions(): boolean | undefined {
```

Comment:
This is not needed for that command since all the command options are specified and you do not have unknown options to specify so you can remove: 

```
public allowUnknownOptions(): boolean | undefined {
    return true;
  }
```